### DISCUSSION TITLE
GroundItems: Change order of operations to handle hidden items first

### DISCUSSION THREAD

**Discussion by nMik6** [runelite/runelite#18160] [Ideas]
When an ground item is highlighted it should first check hidden items instead of the opposite like it currently does.

Assumption:
Highlight = '*seed'
Hidden = 'belladonna seed'

In the current state, the belladonna seed would still be highlighted on the ground because it checks for highlights before that of hidden. In the proposed state, we would check hidden first before determining if we highlight an item on the ground.

**Comment by harbingerofme** [comment]
And now consider the opposite state:
Hidden = "*seed"
Highligt = "Ranarr seed"
Your proposed state would just be as counterintuitive

**Reply by nMik6** [reply]
Valid point, but then how about a toggle to determine which is prioritized instead. That way both use cases can be handled.
